.github/workflows/test-huggingface-dlcs.yml

```diff
@@ -0,0 +1,39 @@
+name: Test Hugging Face DLCs
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - synchronize
+      - ready_for_review
+    branches:
+      - main
+    paths:
+      - tests/*
+      - pytest.ini
+      - .github/workflows/run-tests-action.yml
+      - .github/workflows/test-huggingface-dlcs.yml
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  dlcs-on-cpu:
+    name: Run Hugging Face DLCs Tests on CPU
+    uses: huggingface/Google-Cloud-Containers/.github/workflows/run-tests-action.yml@add-integration-tests
+    with:
+      group: aws-general-8-plus
+      inference-dlc: us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-pytorch-inference-cpu.2-2.transformers.4-44.ubuntu2204.py311
+
+  dlcs-on-gpu:
+    name: Run Hugging Face DLCs Tests on GPU
+    uses: huggingface/Google-Cloud-Containers/.github/workflows/run-tests-action.yml@add-integration-tests
+    with:
+      group: aws-g4dn-2xlarge
+      training-dlc: us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-pytorch-training-cu121.transformers.4-42.ubuntu2204.py310
+      inference-dlc: us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-pytorch-inference-cu121.2-2.transformers.4-44.ubuntu2204.py311
+      tgi-dlc: us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-text-generation-inference-cu121.2-2.ubuntu2204.py310
```

Comment:
Mhm is there a better way to specify those? Feels like we can easily forget updating them?